management/src/main/resources/reference.conf

```diff
@@ -52,7 +52,13 @@ akka.management {
   }
 
   # Health checks for readiness and liveness
+  http.route-providers += "akka.management.HealthCheckRoutes"
+
   health-checks {
+
+    # whether health-check route-providers should be enabled
+    enabled = true
```

Comment:
enabled by default, although I'm not so sure about this. We want it for sure when in combination with Cluster Bootstrap, but maybe not in other cases.

Reply:
We shouldn't make it complicated. Either it is enabled or disabled by default, then the actual liveness and readiness checks are contributed by other modules such as `ClusterMembershipCheck` from akka-management-cluster-http.

I think it's fine to have it enabled by default like this, with the option to disable it.